cogs/coding.py

```diff
@@ -281,7 +291,7 @@ async def documentation(self, ctx: Context, language: str, *, query: str) -> Non
         lang = language.strip("`")  # same thingy again
```

Comment:
289:
D202: No blank lines allowed after function docstring (found 1)
D401: First line should be in imperative mood

Reply:
Same goes for line 302